experiments/dropout/simple_dropout.py

```diff
@@ -61,10 +60,10 @@ def main():
     results = core.swarm_train(bee, num_swarm=20, fields="ypred,loss,y_nodrop,loss_nodrop")
     print("Done Training, starting animating")
     basef = f"{dropout}_dropout_{h}h{w}w_{nepoch}e"
-    animator.make_animation(xt, yt, results["ypred"], basef,
-                            f"data_out/{basef}.mp4")
-    animator.make_animation(xt, yt, results["y_nodrop"], f"Eval {basef}",
-                            f"data_out/eval_{basef}.mp4")
+    animator.make_animation(xt, yt, results["ypred"], basef, f"data_out/{basef}.mp4")
+    animator.make_animation(
+        xt, yt, results["y_nodrop"], f"Eval {basef}", f"data_out/eval_{basef}.mp4"
+    )
     print("Animation done")
```

Comment:
Oh yeah, I use this autoformatting tool called `black` which does this. It's a bit inconsistent right now since doing it can cause merge conflicts if I'm not careful